llvm/docs/LangRef.rst

```diff
@@ -12895,12 +12895,16 @@ vectors of the same size, and the selection is done element by element.
 If the condition is an i1 and the value arguments are vectors of the
 same size, then an entire vector is selected.
 
+Note that violations of poison-generating flags for both arms may not result in poison return values.
```

Comment:
I don't understand this sentence. If we violate poison-generating flags for *both* arms the result will always be poison, right?